parser_library/include/debugger.h

```diff
@@ -70,10 +71,16 @@ class debugger
     debugger& operator=(debugger&&) & noexcept;
     ~debugger();
 
-    bool launch(sequence<char> source, workspaces::workspace& source_workspace, bool stop_on_entry);
-    bool launch(std::string_view source, workspaces::workspace& source_workspace, bool stop_on_entry)
+    void launch(sequence<char> source,
+        workspaces::workspace& source_workspace,
+        bool stop_on_entry,
+        workspace_manager_response<bool> resp);
+    void launch(std::string_view source,
+        workspaces::workspace& source_workspace,
+        bool stop_on_entry,
+        workspace_manager_response<bool> resp)
```

Comment:
From SonarCloud:
```suggestion
    void launch(sequence<char> source,
        const workspaces::workspace& source_workspace,
        bool stop_on_entry,
        workspace_manager_response<bool> resp);
    void launch(std::string_view source,
        const workspaces::workspace& source_workspace,
        bool stop_on_entry,
        workspace_manager_response<bool> resp)
```

Reply:
I know about this and prefer to keep the non-const ref as we might need it in the future.